modules/ROOT/pages/api-groups-modifying-groups.adoc

```diff
@@ -0,0 +1,115 @@
+= Modify API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If your business requirement changes, you might want to modify an API Group by adding new API instances to it. To cover new scenarios, you might want to add a new API instance. 
+
+You can also version an existing instance to add customizations. If a service is no longer required, you can remove the respective API instance that supports the service.
+
+The following tasks are covered in this section:
+
+* <<adding-api-instances-to-groups, Add API instances to Existing API Groups>>
+* <<removing-api-instances-from-groups, Remove API instances from an API Group>>
+* <<creating-new-versions-of-groups, Create new versions of an API Group>>
+
+
+== Restrictions for Modifying API Group and Instances
+
+Before you modify an API Group to add or remove API Group instances, ensure that you carefully reviews the restrictions. You can modify the API Group instance anytime if you meet these restrictions. You can:
+
+* Add new API instances of APIs that already exist in the group only if the API Group instance has no contracts. 
+* Add only those API instances that belong to the same identity provider (IdP) as the API Group instance. You cannot mix IdPs in an API Group instance.
+* Add API instances but not APIs if:
+** The API Group instance is published.
+** The API Group version is published.
+** There are more than one API Group instances of the same API Group version.
+
+== Exchange Status Based on API Group Changes
+
+The status of the API Group in Exchange changes Based on the scenario:
```

Comment:
```suggestion
The Exchange status of an API Group depends on the conditions:
```
But is this really a thing, "an Exchange status," or do we mean simply what is reflected in the UI?

Reply:
Also, this section is not about "changes" but specifically adding a new API instance, so maybe the title needs tweaking.

Reply:
Done! Thanks!